Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -63,8 +132,26 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```

Comment:
InsertFirst should use InsertAt(array, number, 0)

Reply:
InsertFirst(int[] array, int number) using InsertAt(array, number, 0)
Thanks for hints and tips! ( now I understand what did you mean, and how to modify )